### ISSUE TITLE
Console view is broken on master

### ISSUE THREAD

**Issue by tardyp** [buildbot/buildbot#4777]
probably due to webpack PR

angular.js:13424 TypeError: o.keys is not a function
    at Console.matchBuildWithChange (main.module.js:371)
    at Console._onChange (main.module.js:143)
    at angular.js:18953
    at p (angular.js:5824)
    at angular.js:6100

**Comment by rajdeepbharati** [comment]
@tardyp Could you tell me how to replicate this issue?

**Comment by rajdeepbharati** [comment]
console view is broken on https://buildbot.buildbot.net/#/console.
However, it seems to work on my local buildbot macports with version 2.3.1-dev98
![Screenshot 2019-05-27 at 11 30 19 AM](https://user-images.githubusercontent.com/30660439/58398565-14332000-8073-11e9-8a4b-e9c0190bf955.png)

**Comment by tardyp** [comment]
With guanlecoja, we had the "gulp dev proxy" option, which allowed to proxy the api and ws endpoints to arbitrary master from the localhost dev environement. 
This was very practical. Perhaps you can reproduce this with webpack (@p12tic , what do you think?)

**Comment by p12tic** [comment]
I agree that `gulp dev proxy` was very useful and we should implement something like that in current www.

**Comment by yak1ex** [comment]
Similar behaviors occurred at my buildbot instance.
- Seems to be related with #4665 according to the error place (basic logic is the same for coffee and js)
- Using `Object.keys(rev)` instead of `rev.keys()` seems to recover the behavior at my instance.
https://github.com/buildbot/buildbot/blob/a470a5d95d1a7d4a8fcdd173bb1c04893733c87a/www/console_view/src/module/main.module.js#L371
